.storybook/4.SHADOW.stories.mdx

```diff
@@ -115,7 +115,7 @@ As well as the neutral colors for shadow 2 other colors exist that are used for
     style={{
       height: 100,
       backgroundColor: 'var(--color-primary-default)',
-      boxShadow: 'var(--shadow-size-lg) var(--color-primary-shadow',
+      boxShadow: 'var(--shadow-size-lg) var(--color-shadow-primary',
```

Comment:
This change will have to happen when we upgrade to v4 `--color-shadow-primary` isn't present currently. Revert changes
```suggestion
      boxShadow: 'var(--shadow-size-lg) var(--color-primary-shadow',
```

Reply:
Updated https://github.com/MetaMask/metamask-extension/pull/25125/commits/ce1b375551c85b72a97d54226d61f7c3d4b4f884

Reply:
Updated https://github.com/MetaMask/metamask-extension/pull/25125/commits/ce1b375551c85b72a97d54226d61f7c3d4b4f884